l1-contracts/contracts/bridgehub/Bridgehub.sol

```diff
@@ -656,13 +656,35 @@ contract Bridgehub is IBridgehub, ReentrancyGuard, Ownable2StepUpgradeable, Paus
     /// @dev IL1AssetHandler interface, used to undo a failed migration of a chain.
     /// @param _chainId the chainId of the chain
     /// @param _assetId the assetId of the chain's STM
-    /// @param _data the data for the recovery
+    /// @param _data the data for the recovery. The structure for this data is:
+    ///              depositAmount || l2Receiver || chainId || stmData || chainData
```

Comment:
if we are going down this path, maybe also document this in `function bridgeBurn(`? Also, if it wont be too expensive, maybe it makes sense for readability to just make one 

```
struct BridgehubSTMAssetData {
   bytes memory stmData;
   bytes memory chainData;
}
```

So we could always just say that this array encodes this struct without having to constantly copy-paste the description of its elements